agent/exec/container/executor.go

```diff
@@ -76,6 +76,7 @@ func (e *executor) Describe(ctx context.Context) (*api.NodeDescription, error) {
 			NanoCPUs:    int64(info.NCPU) * 1e9,
 			MemoryBytes: info.MemTotal,
 		},
+		Addr: info.Swarm.NodeAddr,
```

Comment:
+1 on manager detection (my bad - I was the one who suggested this way, I didn't think about manager detection).

In that case, the address field should go straight into Node (as NodeDescription is owned by the agent). @stevvooe confirm?